_posts/2017-02-21-release-3-0-0.markdown

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Release 3.0.0"
+date: 2017-02-21 14:00:00
+author: "@jubianchi"
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.0.0][milestone]!
+
+## Stats
+
+* 55 [commits][commits],
+* 580 files changed,
+* 8 contributors,
+* 5 [new features][changelog],
+* 1 [bug fix][changelog].
+
+Let's talk about these numbers. Especially one of them: you might have noticed the 580 files changed. This is a lot! 
+atoum has a total of 557 files (source files and test files) so 580 is really a lot of change but this is for some good 
+reasons. Guess what: **atoum finally adopted a common coding-style, [PSR-2][PSR-2]**.
+
+We hope this will be a good move for two main reasons: lower the friction for new contributors and ease our work when 
+reviewing PRs: we hooked up [StyleCI][StyleCI] and [PHP CS Fixer][PHP CS Fixer].
+
+Enough with numbers, let's talk about new features!
+
+## What's new
+
+**Everything we will talk about here is also available in the [changelog][changelog] and in the [upgrade guide][upgrade].**
+
+### Deprecations and breaking changes
+
+As you may know atoum is release under the [semver][semver] convention. **The 3.0.0 release comes with its breaking 
+changes**. Most of them are minor and you might not notice them but it's woth giving you some details.
+
+#### Runtime
+
+We have **dropped support for some PHP and xDebug versions**. atoum now supports PHP `>= 5.6.0` and xDebug `>= 2.3.0`.
+
+Take care of the xDebug version you use: atoum might work with a version lower than `2.3.0` but you might encounter 
+segfaults on latest PHP versions.
+
+#### Reports
+
+atoum used to provide some fancy reports. You probably known them: the Nyancat and Santa reports. As these reports are 
+not really usefull **they have been moved to the [reports-extension][reports-extension]**. If you
+were using them, install the extension and they should work without requiring you to change anything to your 
+configuration file.
```

Comment:
I will add some link to readme of extension or extension website or the documentation